src/qspdlog_style_dialog.cpp

```diff
@@ -33,23 +37,37 @@ QSpdLogStyleDialog::QSpdLogStyleDialog(QWidget* parent)
         loggerNameEdit,
         &QLineEdit::textChanged,
         this,
-        [ this, backgroundColorEdit, textColorEdit ](const QString& name) {
+        [ this, backgroundColorEdit, textColorEdit, checkBoxBold ](const QString& name) {
         std::string namestdstr = name.toStdString();
         auto bg = _model->getLoggerBackground(namestdstr);
         auto fg = _model->getLoggerForeground(namestdstr);
+        auto fnt = _model->getLoggerFont(namestdstr);
 
         if (bg)
             backgroundColorEdit->setText(bg.value().color().name());
+        else
+            backgroundColorEdit->setText("");
 
         if (fg)
             textColorEdit->setText(fg.value().name());
+        else
+            backgroundColorEdit->setText("");
```

Comment:
You probably meant `textColorEdit`
```suggestion
            textColorEdit->setText("");
```